src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -51,15 +51,16 @@
 		<span v-if="isLoading"
 			v-tooltip="previewTooltip"
 			class="preview loading" />
-		<button v-if="isUploadEditor"
+		<Button v-if="isUploadEditor"
 			tabindex="1"
+			type="primary"
 			:aria-label="removeAriaLabel"
-			class="remove-file primary">
+			class="remove-file">
```

Comment:
Please again remove all CSS rules which don't apply anymore.
I think it's
```
	box-shadow: 0 0 4px var(--color-box-shadow);
	width: $clickable-area;
	height: $clickable-area;
	padding: 0;
	margin: 0;
	&__icon {
		color: var(--color-primary-text);
	}
```

Reply:
Ok